deps/madara/Dockerfile

```diff
@@ -12,12 +10,14 @@ RUN apt-get -y update && \
 # Set the working directory
 WORKDIR /usr/src/madara/
 
-RUN echo Cloning madara from branch ${MADARA_VER}
-RUN git clone --branch ${MADARA_VER} --single-branch https://github.com/madara-alliance/madara . && \
+ENV MADARA_VER="a99088a96a8126d33fef4fabba76a70032e601cf"
```

Comment:
same as in the orchestrator image, why is this needed?

Reply:
Changed to main branch now that the 3 projects are there